local/local.go

```diff
@@ -155,6 +160,27 @@ func picardImage(output io.Writer) (string, error) {
 	return fmt.Sprintf("%s@%s", picardRepo, sha), nil
 }
 
+func getPicardSha(picardVersion string) (string, error) {
+	// If the version was passed as argument, we take it
+	if picardVersion != "" {
+		return picardVersion, nil
+	}
+
+	var sha string
+	var err error
+
+	sha, err = loadBuildAgentShaFromConfig()
+	if sha != "" && err == nil {
+		return sha, nil
+	}
```

Comment:
if `err != nil` then we never inform the user its configuration file is not valid, shouldn't we ? 
either told them that the configuration could not have been red/parsed, and we're using the latest picard version, or crash

Reply:
True, let's add a message